domain-ee/ee-ep-merge-app/src/python_src/service/ep_merge_machine.py

```diff
@@ -274,17 +279,49 @@ def on_get_ep400_contentions(self, event, pending_contentions_response=None):
 
         self.send(event=event, pending_contentions_response=pending_contentions_response, ep400_contentions_response=response)
 
+    def verify_ep400_contention_special_issue_codes(self, ep400_contentions: list[ContentionSummary], pending_contentions: list[ContentionSummary]) -> None:
+        request = get_special_issue_types.Request()
+        response = self.make_request(
+            request=request, hoppy_client=HOPPY.get_client(ClientName.GET_SPECIAL_ISSUE_TYPES), response_type=get_special_issue_types.Response
+        )
+
+        if response and response.status_code == 200:
+            deactive_special_issue_codes = [
+                special_issue.code
+                for special_issue in response.code_name_pairs or []
+                if special_issue.deactive_date and special_issue.deactive_date < datetime.now(special_issue.deactive_date.tzinfo)
+            ]
+            if deactive_special_issue_codes:
```

Comment:
Oof, this method gets really deeply nested. What if you negate the `if`s on lines 288 and here (294) to fail-fast? That will at least save on some indenting.

Still not sure I love the length though. Maybe the more nested code here can come out into a new method?